samigo/samigo-qti/src/java/org/sakaiproject/tool/assessment/qti/helper/ExtractionHelper.java

```diff
@@ -1160,8 +1160,11 @@ public void makeSectionAttachmentSet(SectionFacade section, Map sectionMap)
    */
   public void makeItemAttachmentSet(ItemFacade item)
   {
+	  Set<ItemAttachmentIfc> set = new HashSet<ItemAttachmentIfc>();
+	  item.setItemAttachmentSet(set);
+	  
 	  // if unzipLocation is null, there is no assessment attachment - no action is needed
-	  if (unzipLocation == null) {
+	  if (unzipLocation == null) {  
```

Comment:
Fix extra blank spaces, don't know why I can't remove this from here... maybe because it's a draft?

Reply:
Fixed...but the indentation of these files has no future.